kratos/mpi/utilities/distributed_model_part_initializer.h

```diff
@@ -58,8 +58,10 @@ class DistributedModelPartInitializer
     /// Constructor.
     DistributedModelPartInitializer(
         ModelPart& rModelPart,
-        const int SourceRank)
+        const DataCommunicator& rDataComm,
```

Comment:
What is the reason for not using the `DataCommunicator` from `ModelPart`?

Reply:
It is not yet initialized
-> check the Code, I am calling the ParallelFillComm